apis/r/R/SOMANDArrayBase.R

```diff
@@ -37,6 +37,8 @@ SOMANDArrayBase <- R6::R6Class(
       ## .is_sparse field is being set by dense and sparse private initialisers, respectively
       private$.type <- type                 # Arrow schema type of data
 
+      private$.soma_context <- soma_context() # package-level cache access
+
```

Comment:
We should be picking up the context from `$new()`

Reply:
Good catch, I think you are right that this is redundant too (similar to the one I removed in SOMADataFrame).

Fixed in a632da13